logistics_project/apps/malawi/warehouse/warehouse_view.py

```diff
@@ -16,6 +18,27 @@ class MalawiWarehouseView(ReportView):
     
     show_report_nav = True  # override to hide
 
+    # If overridden to True, implement get_min_start_date() and the start date
+    # for the report will automatically update if the default range or the
+    # user-selected range is too wide.
+    automatically_adjust_datespan = False
+
+    def get_min_start_date(self, request):
+        """
+        Should return the minimum start date that should be used with this report.
+        """
+        raise NotImplementedError()
```

Comment:
only reports that override `automatically_adjust_datespan = True` need to implement it - i added a comment above the `automatically_adjust_datespan` property but can add one here too